assets/sass/protocol/base/elements/_links.scss

```diff
@@ -4,61 +4,50 @@
 
 @use '../../includes/lib' as *;
 
-:link {
-    color: $link-color;
+a {
+    color: var(--link-color);
     text-decoration: underline;
 
-    &:hover,
-    &:focus,
-    &:active {
-        color: $link-color-hover;
+    &:where(:hover, :focus, :active) {
+        color: var(--link-color-hover);
         text-decoration: none;
     }
 
-    &:active {
+    &:where(:active) {
         background-color: rgba(0, 0, 0, 0.05);
     }
-
-    @supports (--css: variables) {
-        color: var(--link-color);
-
-        &:hover,
-        &:focus,
-        &:active {
-            color: var(--link-color-hover);
-        }
-    }
 }
 
-:visited {
-    color: $link-color-visited;
-
-    &:hover,
-    &:focus,
-    &:active {
-        color: $link-color-visited-hover;
-    }
-
-    @supports (--css: variables) {
-        color: var(--link-color-visited);
+a:where(:visited) {
+    color: var(--link-color-visited);
 
-        &:hover,
-        &:focus,
-        &:active {
-            color: var(--link-color-visited-hover);
-        }
+    &:where(:hover, :focus, :active) {
```

Comment:
wiping out specificity of stateful pseudo-classes with `:where()` 
<img width="404" alt="Screenshot 2025-04-16 at 12 37 28 PM" src="https://github.com/user-attachments/assets/ded743dd-1627-483d-900d-2b216c80c23a" />

Reply:
Another thing we could do to make styling links easier would be to drop the separate styles for visited links on hover/focus/active and [rely on source order for the override](https://css-tricks.com/css-basics-styling-links-like-boss/#aa-style-each-link-state).

1.     Link
2.     Visited
3.     Hover
4.     Active
5.     Focus

Reply:
I've always found the :link:visited:hover edge case to be confusing.

Reply:
Will add a commit with source order for comparison

Reply:
The only potential conflict here is that a stateful pseudo-class + element selector will beat a component class selector (if it doesn't also explicitly override the state). This is most likely to happen with `button` component and link-specific `visited` state.

We could unnest the link states so they are only 0,1,0 specificity and rely on source order to override (basic element styles are overruled by component styles)

Or we could also include the `:visited` state selector on any elements that could also be links under the hood